src/components/usa-icon/usa-icon--sizes/usa-icon--sizes.stories.js

```diff
@@ -0,0 +1,17 @@
+import "../../../stylesheets/uswds.scss";
+import Component from "./usa-icon--sizes.twig";
+import Data from "./usa-icon--sizes.json";
+
+export default {
+  title: "Components/Icon Sizes",
+  argTypes: {
+    img_path: {
+      table: { disable: true },
+    },
+  },
```

Comment:
This doesn't seem necessary. I see the same result without this setting.

Reply:
It hides the img_path modifier.

Before:
![image](https://user-images.githubusercontent.com/25211650/151841277-7d904e0b-2264-4f12-98d3-89f8905b470c.png)

After:
![image](https://user-images.githubusercontent.com/25211650/151841443-77d916b8-f619-442c-9465-30ea82b1ef99.png)

I can also add it for `icons` and hide all controls if we think that's better
![image](https://user-images.githubusercontent.com/25211650/151841652-0f9ba67d-1ee0-4840-9ec2-fcc90a61361e.png)

Reply:
Nice! Let's hide them in this case.